app/Models/Bible/StreamBandwidth.php

```diff
@@ -22,6 +22,6 @@ public function transportStreamTS()
 
     public function transportStreamBytes()
     {
-        return $this->hasMany(StreamBytes::class)->orderBy('offset');
+        return $this->hasMany(StreamBytes::class);
```

Comment:
why was the orderBy removed?

Reply:
@bradflood, I added this two days ago when we decided to create a new record in the bible_file_stream_bytes entity to ensure that it would be in the correct order.

However, I removed it in this pull request because the new record had not been created.

I plan to add it again to enforce the proper order when retrieving the bible_file_stream_bytes records.